src/runtime_src/core/include/xclbin.h

```diff
@@ -570,11 +570,13 @@ extern "C" {
         uint32_t mpo_name;                  // Name of the aie_partition 
         uint32_t operations_per_cycle;      // Operations per cycle. Used later to create TOPS (operations_per_cycle * <AIE Clock Frequency>)
         uint8_t padding[4];
+        uint64_t inference_fingerprint;
```

Comment:
Could you add a one line comment description describing what this variable represents.
```suggestion
        uint64_t inference_fingerprint;  // The unique hash value of the inference function
```